src/transformers/models/auto/tokenization_auto.py

```diff
@@ -167,6 +167,7 @@
             ("layoutxlm", ("LayoutXLMTokenizer", "LayoutXLMTokenizerFast" if is_tokenizers_available() else None)),
             ("led", ("LEDTokenizer", "LEDTokenizerFast" if is_tokenizers_available() else None)),
             ("lilt", ("LayoutLMv3Tokenizer", "LayoutLMv3TokenizerFast" if is_tokenizers_available() else None)),
+            ("llama", ("LLaMATokenizer", None)),
```

Comment:
Needs to be protected by an is_sentencepiece_available here (look at hos it's done for BART for instance).